cclib/parser/molcasparser.py

```diff
@@ -40,13 +40,14 @@ def normalisesym(self, label):
     def before_parsing(self):
         # Compile the regex for extracting the element symbol from the
         # atom label in the "Molecular structure info" block.
-        self.re_atomelement = re.compile('([a-zA-Z]+)\d+')
+        self.re_atomelement = re.compile('([a-zA-Z]+)|d+')
```

Comment:
Was this supposed to be changed? Or are you missing a `\` for the a digit class?

Reply:
So it should really be, I think `([a-zA-Z]{1,3})\d*` which I check and does work. But, I really don't think we need regexes for this. Rewrote this Pythonically in e0c65498759d0f5955e19904cd6202ae1b6c4302.